cmd/clusterctl/pkg/client/config/providers_client.go

```diff
@@ -90,6 +90,11 @@ func (p *providersClient) defaults() []Provider {
 			url:          "https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/latest/infrastructure-components.yaml",
 			providerType: clusterctlv1.InfrastructureProviderType,
 		},
+		&provider{
+			name:         "openstack",
```

Comment:
What about creating a "OpenStackProviderName const? We are ending up in creating constants for all the provider name...

Reply:
yes, I made update accoridngly, thanks